core/templates/dev/head/pages/exploration_editor/switch_editor_directive.html

```diff
@@ -0,0 +1,38 @@
+<script type="text/ng-template" id="inline/switch_editor_directive">
+  <div class="switch-editor-btn-container">
+    <button type="button" class="btn btn-default oppia-switch-editor-btn" ng-click="setEditorModeToFull()" ng-if="isEditorInSimpleMode()">
+      Use Advanced Editor
+    </button>
+    <button type="button" class="btn btn-default oppia-switch-editor-btn" ng-click="setEditorModeToSimple()" ng-if="isEditorInFullMode()">
+      Use Simple Editor
+    </button>
+  </div>
+  <style>
+  .switch-editor-btn-container {
```

Comment:
Prefix all classes in this file with .oppia-

Also prefix all CSS rules with `editor-switch-button` so that they apply only to this directive. You can then use shorter class names within the directive itself.